src/utils/github/getProjectsApi.js

```diff
@@ -1,21 +1,43 @@
-import { Octokit } from "@octokit/rest";
+import { graphql } from "@octokit/graphql";
 import extractOwnerRepo from "./extractOwnerRepo";
 
 export default async function getProjectsApi(url, token) {
   // get owner and repo name from url
   const { owner, repo } = extractOwnerRepo(url);
 
   // get data from github api using user's API
-  const octokit = new Octokit({
-    auth: token,
+  const octokit = graphql.defaults({
+    headers: {
+      authorization: `token ${token}`,
+    },
   });
   let response;
   try {
-    response = await octokit.rest.projects.listForRepo({
-      owner,
-      repo,
-      state: "open",
-    });
+    response = await octokit(
+      `
+      query($owner: String!, $repo: String!) {
+        repository(owner: $owner, name: $repo) {
+          projectsV2(first: 10) {
+            nodes {
+              id
+              title
+              closed
+            }
+          }
+        }
+      }
+      `,
+      {
+        owner,
+        repo,
+      },
+    );
+    console.log(response);
+    console.error(response);
```

Comment:
I don't think this debug code is required anymore

```suggestion

```

Reply:
True 😅